### ISSUE TITLE
NPM release to resolve security vulnerability

### ISSUE THREAD

**Issue by cdrini** [less/less-plugin-clean-css#29]
The version of the clean-css dependency has been updated to a safe version, but the version of less-plugin-clean-css on npm is outdated and doesn't have the fix. Could we release a new version on npm?

**Comment by thesocialdev** [comment]
+1

**Comment by ZLevine** [comment]
Yes, please publish the latest version—I can't use this library because we need clean-css V4.

**Comment by BasixKOR** [comment]
Any news on this?

**Comment by thesocialdev** [comment]
Just FYI, we couldn't wait for this to be published and published in our org. See https://www.npmjs.com/package/@wikimedia/less-plugin-clean-css
